about.html

```diff
@@ -0,0 +1,166 @@
+    <!-- MAIN STARTS -->
+
+    <main>
+      <section class="main-opening about-page container">
+        <p class="intro about-hide">Adopting Global Best Practices in</p>
+        <h1 class="about-header">Community Pharmacy Practice</h1>
+        <hr />
+        <p class="description about-description">
+          Attended by most major manufacturers, distributors and community
+          Pharmcists in the Pharmacy space, the ACPN conference provides a
+          focused and unique opportunity to learn, share best practices, network
+          with peers, and interact with other healthcare providers in the
+          pharmaceutical space. At the ACPN Conference, community Pharmacists
+          can learn how best to offer pharmaceutical care, gain insights on more
+          efficient management processes and become aware of new technology and
+          techniques, which will ultimately help them and their companies: Sell
+          more products, Reduce costs, and Increase efficiencies
+        </p>
+        <p class="about-venue">
+          Please contact us via email for any further question and
+          clarifications about the ACPN conference 2023
+        </p>
+        <a class="email" href="#">info@acpn2023.org</a>
+      </section>
+    </main>
+    <!-- MAIN ENDS -->
+
+    <!-- SUB-SECTION STARTS -->
+    <section class="ending">
+      <div class="holder">
+        <h2>ACPN Conference 2023 Logo</h2>
+        <hr />
+        <p>
+          The Logo of the ACPN Conference 2023 will be decided through the
+          annual ACPN logo competition on June 20, 2023.
+        </p>
+        <h4 class="acpn">
+          ACPN NATIONAL <br />
+          CONFERENCE 2023.
+        </h4>
+      </div>
+      <hr class="dash" />
+      <div class="holder">
+        <h2>See the past ACPN Conference</h2>
+        <hr />
+        <p>
+          Take a look at the last two ACPN Conference which took place in Abuja,
+          Nigeria.
+        </p>
+
+        <div id="meeting">
+          <div class="meeting-start">
+            <h3 class="meeting-header">2021</h3>
+            <p class="meeting">ACPN Conference 2021 in Abuja</p>
+          </div>
+
+          <div class="meeting-middle">
+            <h3 class="meeting-header">2019</h3>
+            <p class="meeting">ACPN Conference 2021 in Enugu.</p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- SUB-SECTION ENDS -->
+
+    <!-- PARTNER STARTS -->
+    <section class="partner">
+      <div class="container">
+        <h4>Partners</h4>
+        <hr />
+        <div class="company-logo home-logo">
+          <p class="sponsor1">ASCON</p>
+          <img src="images/Alpha-Logo.jpeg" alt="" />
+          <img src="images/May-Baker-Logo.png" alt="" />
+          <img src="images/Oakleaf-Logo.jpeg" alt="" />
+          <img src="images/sanofi.jpg" alt="" />
+          <img src="images/Bydow-Logo.png" alt="" />
```

Comment:
- Good job done so far, but to help improve the semantics of your page, I suggest wrapping this section in a `ul` tag and also wrap each image in a corresponding `li` tag. Please fix this 🙏 
- Also, in the mobile section of this page, please remember to stick to the spacing, margins, borders as in the design template. I suggest improving the margins between each image. Please fix this 🙏 

Your project:
![egaerg](https://user-images.githubusercontent.com/73607512/234027654-de68b4cb-80a0-4216-bd7d-572b83e33a5e.PNG)

Design template:
![wrgawrgqr](https://user-images.githubusercontent.com/73607512/234027845-b599dd09-9bc1-42dc-8c07-5cd0b9ffebd3.PNG)